src/main/java/pooh/Deadline.java

```diff
@@ -0,0 +1,110 @@
+package pooh;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * Represents a deadline task that extends the Task class.
+ * <p>
+ * The Deadline class is a subclass of the Task class, and it adds two additional attributes:
+ * deadlineString and deadlineDateTime to represent the deadline of the task.
+ * </p>
+ */
+public class Deadline extends Task {
+    private final String deadlineString;
```

Comment:
I like how you named the variables here. They maintain the coding standard and are easy to understand at first glance what they represent.